### DISCUSSION TITLE
Support for custom pages / imprint, privacy protection, refund policy

### DISCUSSION THREAD

**Discussion by matserix** [Paymenter/Paymenter#260] [Ideas]
Support custom page creation...

For imprint, privacy policy, could just be like tos for gdpr

**Comment by finnjosten** [comment]
Tbh custom pages in general would be nice. Would be kinda hard to make
 though

**Comment by GreenArrowlol** [comment]
really need this + a dropdown button beside Announcements would be great for custom pages.
